### PR TITLE
Include `load` in summaries

### LINKED ISSUE #95: Show example load() call for each rule
Whenever I start using a new Starlark rule, the first step is to `load()` it in my BUILD file. Therefore it would be handy if the Stardoc output for each rule showed an appropriate `load()` statement that I could directly copy-paste. That way, users don't have to manually tweak the path to the `.bzl` file, change the final slash to a colon, etc., which is pure avoidable tedium.

### PR DESCRIPTION
**Pull request by fmeum**
Rules, providers, functions and aspects now have a `load` statement in their summary. Aspects additionally include a copyable `--aspects` flag.

Fixes #95

### CODE REVIEW

**Comment by fmeum** [comment]
Some tests are failing due to differing repository names. I can look into this when there is general consensus on the approach.

**Review by tetromino — CHANGES_REQUESTED** [review]
Looks good! I updated the branch after the recent churn, which hopefully should fix test failures.

One nit: I would put a blank line between `load(...)` and the usage of the symbol, as in buildifier-formatted output.

File: src/main/java/com/google/devtools/build/stardoc/rendering/MarkdownUtil.java
[the remark this answers is not in the document]
Comment (fmeum) [reply src/main/java/com/google/devtools/build/stardoc/rendering/MarkdownUtil.java:1]: I did that. I got rid of the aspect flag comment as it feels to niche.

**Comment by fmeum** [comment]
The WORKSPACE tests are failing since they use a different repo name for Stardoc, which now appears in `load` statements. How do you want me to deal with that?

**Comment by tetromino** [comment]
> The WORKSPACE tests are failing since they use a different repo name for Stardoc, which now appears in `load` statements. How do you want me to deal with that?

Excellent question! We may have to tag those tests "manual" + bazel version for which the golden was generated; in #232, I am adding a way to auto-update tests whose goldens are valid only for a particular Bazel release.

**Comment by fmeum** [comment]
I'll wait for #232 then, let me know what I need to do after it has been merged.

**Review by tetromino — APPROVED** [review]
LGTM.

I took the liberty to fix up the tests, since the merge + necessary tagging and bazelci setup is a bit tricky.

**Pull request merged**